mmv1/products/compute/NetworkEdgeSecurityService.yaml

```diff
@@ -0,0 +1,110 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+!ruby/object:Api::Resource
+name: 'NetworkEdgeSecurityService'
+min_version: beta
```

Comment:
Because there are few things being moved from alpha do beta and from beta to v1. For example: the ddos_protection = ADVANCED_PREVIEW in "ComputeRegionSecurityPolicy" is not getting affected in v1 yet. I have started working in other things related to these resources already and I as move forward I will keep looking at the best time do bump to g.a. WDYT?

Reply:
Got it! Sounds good. We can add this to beta first and promote it to ga later. Thanks for the explanation